packages/dashboard/src/hooks/useAssetDescriptionQueries.ts

```diff
@@ -17,16 +17,15 @@ export const useAssetDescriptionMapQuery = (siteWiseAssetQuery: SiteWiseAssetQue
   const { iotSiteWiseClient } = useClients();
 
   const fetchSiteWiseAssetQueryDescription = async () => {
-    if (!iotSiteWiseClient || !siteWiseAssetQuery) return;
-
+    if (!iotSiteWiseClient || !siteWiseAssetQuery) return [];
```

Comment:
returning undefined erroring and clogging up the console 
<img width="542" alt="Screenshot 2023-06-16 at 08 27 48" src="https://github.com/awslabs/iot-app-kit/assets/28601414/aacb27a4-81b7-4a2f-aaff-4601714599b5">